### PR TITLE
Upasthiti

### PR DESCRIPTION
**Pull request by Ridhima10**
Hello Pa
issue resolved- #4142

### CODE REVIEW

**Comment by Ridhima10** [comment]
Screenshot 2024-08-02 190008

File: src/data/userData.tsx [hunk 1]
```diff
@@ -432,5 +432,14 @@ export const Users: User[] = [
       website: "https://ridhima10.github.io/Smart_Tv_Clone/",
       source: "https://github.com/Ridhima10/Smart_Tv_Clone",
       tags: ["html", "css", "javascript", "opensource"],
+    },
+    // upasthiti
+    {
+      title: "Upasthiti",
+      description: "Online attendance taking website which is made up of html, css , javascript, nuxt and vue js",
+      preview: require("./showcase/upasthiti.png"),
+      website: "https://ridhima10.github.io/Upasthiti-frontend/",
```
Comment (Ajay-Dhangar) [inline src/data/userData.tsx:441]: image

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
Hello, the URL provided does not seem to be correct, as it only shows the content of your README.md file, not the rendered web or source code. Please check the URL and make sure it points to the correct location. Thank you!

**Comment by Ridhima10** [comment]
@Ajay-Dhangar okay sure I'll check and correct it if needed!!

**Pull request closed without merging**